nextcord/http.py

```diff
@@ -130,6 +130,15 @@ def _modify_api_version(version: Literal[9, 10]):
     Route.BASE = f"https://discord.com/api/v{version}"
 
 
+def _get_logging_auth(auth: str | None) -> str:
+    if auth is None:
+        return "None"
+    elif len(auth) < 12:  # This shouldn't ever occur, but whatever.
+        return "[redacted]"
+    else:
+        return f"{auth[:12]}[redacted]"
```

Comment:
```suggestion
    else:
        return f"{auth[:12]}[redacted]"
```

out of bounds string slicing returns an empty string:
```py
>>> "hello"[:12]
'hello'
>>> ""[:12]
''
```